airflow/www/utils.py

```diff
@@ -440,6 +440,11 @@ def dag_run_link(attr):
     return Markup('<a href="{url}">{run_id}</a>').format(url=url, run_id=run_id)
 
 
+def task_try_count(ti: TaskInstance) -> int:
+    """Get the total try count for a task instance"""
+    return ti._try_number if ti._try_number != 0 or ti.state in State.running else ti._try_number + 1
```

Comment:
how come `ti.try_number` doesn't give the right answer?

Reply:
I'm not really sure. It seems to cause another issue: https://github.com/apache/airflow/issues/24958
Happy to hop on a call an investigate more.

Reply:
I think it's an issue with running task vs non running tasks

https://github.com/apache/airflow/blob/0ec9651f88b7073f8de4df70b21e70be0bd08b2a/airflow/models/taskinstance.py#L611-L623

It also effected total try count
https://issues.apache.org/jira/browse/AIRFLOW-2143
https://github.com/apache/airflow/blob/9514b6ca701728bc21c609af18a59a2ba9cffe97/airflow/www/views.py#L3237-L3240

Reply:
confusing

Reply:
We should add a new property on TaskInstance for this logic. Giving it a good name and documenting both it and `try_number` would help things.